README.md

```diff
@@ -174,6 +174,49 @@ yarn build:integration
 yarn test:integration
 ```
 
+## Branching Model and Releases
+
+<!-- TODO: explain about changesets + how we do npm publishing + docker publishing -->
+
+### Active Branches
+
+| Branch          | Status                                                                           |
+| --------------- | -------------------------------------------------------------------------------- |
+| [master](https://github.com/ethereum-optimism/optimism/tree/master/)                   | Does not accept PRs. Changes are force-pushed from `develop` when we intend to deploy to mainnet.            |
+| [develop](https://github.com/ethereum-optimism/optimism/tree/develop/)                 | Accepts PRs that are compatible with `master` OR from `regenesis/X.X.X` branches                                |
+| regenesis/X.X.X                                                                        | Accepts PRs for all changes, particularly those not backwards compatible with `develop` and `master`. |
+
+### Overview
+
+We generally follow [this Git branching model](https://nvie.com/posts/a-successful-git-branching-model/).
+Please read the linked post if you're planning to make frequent PRs into this repository (e.g., people working at/with Optimism).
+
+### The `master` branch
+
+The `master` branch contains the code for our latest "stable" releases.
+Updates from `master` always come from the `develop` branch.
+We only ever update the `master` branch after code within the `develop` branch has been successfully deployed to the Optimistic Ethereum mainnet.
+Our update process typically takes the form of a forced push of the `develop` branch into the `master` branch.
```

Comment:
We should be merging `develop` into `master`. See the nvie branching model link.